packages/truffle-core/lib/run.js

```diff
@@ -54,9 +54,9 @@ const Run = {
   },
 
   // executes command or throws user helpful error
-  run(pluginConfigs, customCommand, config) {
-    let runCommand = this.initializeCommand(pluginConfigs, customCommand);
-    runCommand(config);
+  run(pluginConfigs, customCommand, config, done) {
+    const runCommand = this.initializeCommand(pluginConfigs, customCommand);
+    runCommand(config, done);
```

Comment:
We don't need the promise check? That surprises me because I'd expect that nothing would be calling `done` in that case

Reply:
🔧